### ISSUE TITLE
Consider to use new translatable strings for the Content tab traffic light

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#462]
See related: https://github.com/Yoast/wordpress-seo/issues/4055
See also the related PR https://github.com/Yoast/wordpress-seo/pull/4297

The traffic light tab link needs text to report the current SEO score, the related PR is currently concatenating two strings:
- a fixed part "Content:" 
- the SEO score string from the "Assessor presenter" config `presenter.js`

I think it is not too bad, since the first string ends with a colon. By the way, concatenation should be avoided when it comes to translatable strings. Maybe a better solution would be adding new, extended, strings to `presenter.js` e.g.:
```
"Content: Focus keyword not set"
"Content: Bad SEO score"
"Content: OK SEO score"
"Content: Good SEO score"
```
These new strings would make translators happy. On the other hand, it is very likely they would be used in just one place.
I don't have a strong opinion about this, I think the first solution is not ideal but could work, the second one is better but maybe overkill. Would appreciate some l10n thoughts :)
